tst/EnergyPlus/unit/FileSystem.unit.cc

```diff
@@ -108,6 +108,52 @@ TEST(FileSystem, getAbsolutePath)
     EXPECT_FALSE(absPathName.find(currentDirWithSep) != std::string::npos); // Make sure "./" doesn't appear in absolute path
 }
 
+TEST(FileSystem, getFileType)
+{
+    fs::path filePath1("./schedulefileA.CSV");
+    auto ext = EnergyPlus::FileSystem::getFileType(filePath1);
+    EXPECT_TRUE(EnergyPlus::FileSystem::FileTypes::CSV == static_cast<EnergyPlus::FileSystem::FileTypes>(ext));
```

Comment:
`getFileType` is already returning a `FileTypes` instance, you don't need this `static_cast` at all.  You can also make use of the enum comparison function like this if you want: 
```
EXPECT_TRUE(compare_enums(EnergyPlus::FileSystem::FileTypes::CSV, ext));
```

Reply:
Thanks! The code has been updated by removing the static cast conversions. BTW, I also tried the `compare_enums()` way you suggested (actually first tried this) but it seems to have some problems in recognizing the function `compare_enums()`--although I can see it is defined in `fixtures/EnergyPlusFixuture.hh`. I tried a few combinations by including fixutures/EnergyPlusFixture.hh, and also changing test parameter from `FileSystem` to `EnergyPlusFixture` but none of them would get the function recognized properly here. So finally just stay with the simpler way of removing the unnecessary static cast.